lib/smoke_test.rb

```diff
@@ -0,0 +1,61 @@
+require 'net/http'
+
+class SmokeTest
+  def initialize(urls)
+    @@urls = urls
+    secrets = Rails.application.secrets
+    @@api_token = secrets.smoke_test[:api_token] || ""
+    @@slack_uri = secrets.smoke_test[:slack_uri]
+    @@slack_token = secrets.smoke_test[:slack_token]
+    @@slack_room = secrets.smoke_test[:slack_room]
+    @@slack_username = secrets.smoke_test[:slack_username]
+    @@slack_emoji = secrets.smoke_test[:slack_emoji]
```

Comment:
You don't probably need to use all these class variables.
Even if we won't inherit anything from this class, for now, these variables can be dangeroues in many ways if you will go to extend this gem at some point.
So I would just use normal instance variables with just one `@`

Reply:
Many thanks @Levia!